pkgs/development/tools/cdecrypt/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
```

Comment:
Better readability and cleaner diffs in the future if one entry per line.

```suggestion
{ lib
, stdenv
, fetchFromGitHub
}:

stdenv.mkDerivation rec {
```

Reply:
change applied in rebased commit.